lib/reverse_words.rb

```diff
@@ -21,7 +21,7 @@ def reverse_words(my_words)
   my_words.length.times do |x|
     my_words[x] = new_string[x]
   end
-
+  return my_words
```

Comment:
Since you're updating the characters in the input parameter my_words in place, you don't need to return anything. Line 24 could simply be `return`.

Reply:
Ditto with line 42